### PR TITLE
fix: allow default and overridable envars in the docker compose file

### PR DESCRIPTION
**Pull request by gooseops**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# These are defaults and you should change them
```
Comment (0xkr8os) [inline .env:1]: does the compose file automatically pick up this file?

Comment (gooseops) [reply .env:1]: yes, tested that

Comment (gooseops) [reply .env:1]: It was the recommended way for achieving default values with overidable environment variables

Comment (gooseops) [reply .env:1]: You can use `docker compose config` to confirm functionality

Comment (0xkr8os) [reply .env:1]: nice

**Review by 0xkr8os — APPROVED** [review]

**Pull request merged**